### DISCUSSION TITLE
docs/guides/multiple-schema

### DISCUSSION THREAD

**Discussion by giscus** [zenstackhq/zenstack-docs#286] [Comments] [answered]
# docs/guides/multiple-schema

ZenStack offers increased flexibility in managing your schema file through multiple schema files and model inheritance.

https://zenstack.dev/docs/guides/multiple-schema

**Comment by Eliav2** [comment, marked as answer]
normally when running `zenstack generate` Zenstack would check for `schema.zmodel` file on current directory, if you want to specify other path, like in the provided example the schema lays in `zmodel/schema.zmodel`, you should specify path using the `--schema` option: `zenstack generate --schema zmodel/schema.zmodel`
